src/ec/suite_b/ecdsa.rs

```diff
@@ -50,6 +66,12 @@ enum ECDSAVerificationAlgorithmID {
     ECDSA_P384_SHA384_FIXED,
 }
 
+impl PartialEq for ECDSAVerificationAlgorithm {
+    fn eq(&self, other: &Self) -> bool { self.id == other.id }
+}
```

Comment:
> Well, there is PartialEq implementation for ECDSASigningAlgorithm

Yes, but that's because we've not yet found a solution for creating a unifying `ring::signature::SigningAlgorithm` interface.

However, we already have `ring::signature::VerificationAlgorithm`, and ultimatenly for these comparison operators to be most useful, we need to be able to compare two `VerificationAlgorithm`s, including comparing an RSA algorithm to an ECDSA algorithm. That means that `ring::signature::VerificationAlgorithm` is what ultimately needs to implement `Eq`/`PartialEq`. Let's try to find a way to do that. For now, I'll remove this part.